### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by NeverovDS**
Implement sort.Interface interface for People type.

### CODE REVIEW

**Comment by burov** [comment]
Please name PR with reasonable names

File: SortGoodV.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	Len() int
+	Less(i, j int) bool
+	Swap(I, j int)
+	}
+
+type Person struct {
+	firstName string
+	lastName  string
+	birthDay  time.Time
+	}
```
Comment (burov) [inline SortGoodV.go]: bad format

File: SortGoodV.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+type People []Person
+
+func (p People) Len() int {
+	return len(p)
+	}
```
Comment (burov) [inline SortGoodV.go]: bad format

File: SortGoodV.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		return false
+	}
+	if p[i].lastName < p[j].lastName {
+		return true
+	}
+	if p[i].lastName > p[j].lastName {
+		return false
+	}
+	return p[i].birthDay.Sub(p[j].birthDay) > 0
+	}
```
Comment (burov) [inline SortGoodV.go]: bad format

File: SortGoodV.go [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	if p[i].firstName > p[j].firstName {
+		return false
+	}
+	if p[i].lastName < p[j].lastName {
+		return true
+	}
+	if p[i].lastName > p[j].lastName {
+		return false
+	}
+	return p[i].birthDay.Sub(p[j].birthDay) > 0
```
Comment (burov) [inline SortGoodV.go]: main sorting criteria is a birthDay not a first name

File: SortGoodV.go [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+	}
+func (p People) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+	}
```
Comment (burov) [inline SortGoodV.go]: bad format

Comment (NeverovDS) [reply SortGoodV.go]: all fix

**Comment by imatsiash-ib** [comment]
@NeverovDS Please use more obvious names for PRs.

File: SortGoodV.go [hunk 6]
```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"time"
+)
+
+type Interface interface {
```
Comment (imatsiash-ib) [inline SortGoodV.go]: This interface is already defined within the `sort` package. Why we need to redefine it ?

Comment (NeverovDS) [reply SortGoodV.go]: rightly

**Pull request closed without merging**